content/200-concepts/100-components/02-prisma-client/080-null-and-undefined.mdx

```diff
@@ -10,9 +10,12 @@ preview: false
 Prisma Client differentiates between `null` and `undefined`:
 
 - `null` is a **value**
-- `undefined` means **do nothing**
+- `undefined` means **no value has been assigned**
 
-> **Note**: This is particularly important to account for in [a **Prisma with GraphQL context**, where `null` and `undefined` are interchangeable](#use-case-null-and-undefined-in-a-graphql-resolver).
+> **Notes**:
+
+- In patch release [3.11.1](https://github.com/prisma/prisma/releases/tag/3.11.1) we have changed what data is returned when filtering MongoDB documents on undefined fields. Previsouly, undefined fields were by default treated the same as null fileds. With the release of version 3.11.1, now **undefined fields are excluded by default** unless explicitly filtered for, using the `iSet` operator. This allows you to query for undefined and null values separately. For more information about filtering for undefined fields, see [Composite types](/concepts/components/prisma-client/composite-types).
```

Comment:
```suggestion
- In [3.11.1](https://github.com/prisma/prisma/releases/tag/3.11.1), we have changed what data is returned when filtering MongoDB documents on `undefined` fields. Previously, `undefined` fields were by default treated the same as null fields. 

With `3.11.1`, now **undefined fields are excluded by default** unless explicitly filtered for, using the `isSet` operator. This allows you to query for undefined and null values separately. For more information about filtering for `undefined` fields, see [Composite Types](/concepts/components/prisma-client/composite-types).
```

Reply:
Maybe also worth adding something like... Unlike SQL databases, MongoDB can store both `undefined` and `null` values. 

To allow you to query for both types, in 3.11.1, we changed what data is returned when filtering MongoDB documents on `undefined` fields.  ...